README.org

```diff
@@ -114,63 +49,133 @@ A typical journal entry for a day would look like this:
   ** TODO teach org-journal how to brew coffee                   :org-journal:
 #+END_SRC
 
-** Changelog
+** Installation
+
+   =org-journal= is available through [[http://marmalade-repo.org/][marmalade]], [[http://melpa.milkbox.net/][melpa]] and [[http://melpa-stable.milkbox.net/][melpa-stable]]. So installation should be
+   trivial:
+
+#+BEGIN_EXAMPLE
+    C-x package-install org-journal
+#+END_EXAMPLE
+
+   Then add =(require 'org-journal)= to your =.emacs=.
+
+** Quickstart
+
+   Doing =C-x org-journal-new-entry= (or =C-c C-j=) will immediately create a journal directory in the
+   default path (customized using the =org-journal-dir= variable), open a file in =org-journal-mode= and
+   insert a journal entry template.
```

Comment:
"...open or create a file...", "...and insert a template for a new journal entry."